src/ChainRules.jl

```diff
@@ -22,6 +22,12 @@ import ChainRulesCore: rrule, frule
 # Experimental:
 using ChainRulesCore: derivatives_given_output
 
+if isdefined(Base, :stack)
+    using Base: stack
+else
+    using Compat: stack
+end
```

Comment:
This should work, will be `===` the one in Base on 1.9:
```suggestion
using Compat: stack
```